### DISCUSSION TITLE
他人とセッションを共有したい！

### DISCUSSION THREAD

**Discussion by pankona** [mobu-of-the-world/mobu#78] [Ideas]
いまの mobu は、基本的には一人用のアプリである。
なんだけど、mobu は普通多人数で用いるものであるので、状況を参加者の間で共有できるようにしたいです。
よろしくおねがいします。

**Comment by kachick** [comment]
https://github.com/mobu-of-the-world/mobu/discussions/48#discussioncomment-537234

日本語は使わないのでは・・・？ 👀 

https://github.com/yoshimaru46/planning-poker
https://planning-poker-46.web.app/

@yoshimaru46 さんのこのアプリケーションが凄い体験良いので、同じ様なことをやりたいという認識でいます！ やり方わからないけど賛成です！ 😄
